[Caffe2] Move tanh function to math

xiaomengy opened the pull request:
Summary: Move tanh function to math

Differential Revision: D8794745

houseroad reviewed (approved):
LGTM! Thanks for cleaning the code.

houseroad commented:
Let's wait for the CI...

xiaomengy committed:
Move tanh function to math (#9328)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/9328

Move tanh function to math

Differential Revision: D8794745

fbshipit-source-id: 3bb01480e02d690b416edc1652f559e4fbb8fdb5
